hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMPerformanceMetrics.java

```diff
@@ -140,4 +142,8 @@ public MutableRate getGetKeyInfoResolveBucketLatencyNs() {
   public void setForceContainerCacheRefresh(boolean value) {
     forceContainerCacheRefresh.add(value ? 1L : 0L);
   }
+
+  public MutableRate getCheckAccessLatencyNs() {
```

Comment:
Thanks @tanvipenumudy for the patch.  Not sure about passing the _private_ metric `MutableRate` variable from this class to an external caller.  Should just have _getters_ and `setters` for the private `checkAccessLatencyNs `variable.  Can we not structure this so the variable remains hidden and private and just declare a setter for `checkAccessLatencyNs`, such as `public void setCheckAccessLatencyNs(long latencyInNs) `and pass the setter to a _**similar**_ utility function to `captureLatencyNs` that sets the `MutableRate` with the setter?

Reply:
Sure @neils-dev, thank you for the review - will be incorporating the aforementioned changes.